### ISSUE TITLE
Date not read correctly in timeline view

### ISSUE THREAD

**Issue by vedkat** [acidb/mobiscroll#523]
Hi,

I found an accessibility issue in the timeline view.

Dates are not read correctly by screenreader NVDA (in German), so that users don't know the correct date.

Steps to repuduce it:

- Go to timeline view
- Tab to an event with a day of month > 9th.
- The day of month will be read as year: For example 10th is read as 2010, 22nd is read as 2022.

I use NVDA (German). The bug is not visible in the speech viewer, but hearable in the acustic output.
Tested on Firefox and Edge.

Thanks for fixing.

**Comment by dioslaska** [comment]
Fixed in 5.28.0
